### ISSUE TITLE
Move five stuck SACs

### ISSUE THREAD

**Issue by tadhg-ohiggins** [GSA-TTS/FAC#3764]
Part of the follow-up from [[Production ssh access]: move “stuck” submission to `disseminated` · Issue #3568 · GSA-TTS/FAC](https://github.com/GSA-TTS/FAC/issues/3568) included [Add submission status filter to Django admin](https://github.com/GSA-TTS/FAC/pull/3660), and when that went into production we discovered that there were five `SingleAuditChecklist` objects with a `submission_status` value of `submitted`; that number should have been zero.

The `SingleAuditChecklist` objects had the following `report_id` values:

*   2023-09-GSAFAC-0000038716
*   2023-06-GSAFAC-0000034013
*   2023-06-GSAFAC-0000005723
*   2021-09-GSAFAC-0000004846
*   2023-06-GSAFAC-0000003498

These five audits have this in common: none of them have any federal awards, and each has a “total amount expended” of zero.

Our validation doesn’t currently catch this, which is why users were able to submit these audits. However, our dissemination code expects submissions to have federal awards, so it fails when handling these submissions, which is why these have `submitted` status.

Our UI shows “Accepted” as the status in the audits table on the user page listing their audits when the status is either `submitted` or `disseminated`, so it appears to the user as if their submission was valid and successful. The only way they would know it wasn’t is by looking for it in dissemination-side search, where it would not appear.

The current consensus seems to be that we should not disseminate these submissions without discussing it with the submitters and determining why they submitted audits that have no awards; by definition, zero awards and a total amount expended of zero fall under the threshold required to make a submission.

There may be some edge case we do not yet understand or know about that means these submissions should be accepted; it’s also possible that these submissions were made in error.

None of these submissions shares a UEI with any other submission in our system except one, and in that case the other submission has been at the Auditor Certified step for quite some time and the evidence suggests that the other submission had been abandoned in favor of the one that is now stuck.

This strongly suggests that none of these submissions have been superseded by a later one—so at this point this is not a question concerning resubmission.

Three of these submissions are from last November, one is from late March 2024, and one is from April 2024. If we end up disseminating some of them, we may need to adjust dates and/or communicate with the relevant agencies about the dates of submission.

Extracting contact details from General Information JSON:

```sh
pbpaste|jq  '{eename: .auditee_name, eecname: .auditee_contact_name, eeemail: .auditee_email, orname: .auditor_firm_name, orcname: .auditor_contact_name, oremail: .auditor_email}|"\(.eename); \(.eecname), \(.eeemail)|\(.orname); \(.orcname), \(.oremail)"'
```

```[tasklist]
- [x]   Determine whether any of the UEIs for these submissions were used in other submissions that had been disseminated (answer: no).
- [x]   [Discover that the actual problem is that these submissions lack a `federal_awards` property](https://github.com/GSA-TTS/FAC/issues/3765) in their JSON, and `intake_to_dissemination.py:IntakeToDissemination.load_federal_award()` doesn't like that.
- [ ] https://github.com/GSA-TTS/FAC/issues/3790
- [ ]   The following steps await what we learn from contacting the folks involved; some apply only if we determine that any of the submissions should be disseminated with no federal audits.
- [ ]   (PENDING FURTHER DISCOVERY) Fix the intake code to allow this, or change validation to prevent submissions with this issue from getting this far, and/or tweak the data for these specific submissions to allow them to be disseminated. This should probably be its own issue.
- [ ]   (PENDING FURTHER DISCOVERY) Run the `.disseminate()` method on each of them.
- [ ]   (PENDING FURTHER DISCOVERY) Verify that each of them has the correct `submission_status` in Django admin.
- [ ]   (PENDING FURTHER DISCOVERY) Verify that the number of results displayed when using the `SingleAuditChecklist` **Submitted** filter in Django admin is zero.
- [ ]   (PENDING FURTHER DISCOVERY) After the appropriate period to allow for cache population, verify that each of them is available on the dissemination side.
- [ ]   (PENDING FURTHER DISCOVERY) Revisit the way we collapse both `submitted` and `disseminated` statuses into “Accepted” in the user table of their audits.
- [ ]   (PENDING FURTHER DISCOVERY) Either change our validation to disallow submissions with no awards or change dissemination to allow them, depending on what we discover.
```

**Comment by James-Paul-Mason** [comment]
All of the reports listed above have been resubmitted and disseminated EXCEPT for the 4th on the list...

Original Report ID: 2021-09-GSAFAC-0000004846 (stuck in "Submitted" status)
Entity UEI: QN8HF1D7C9L8
Second Report ID: 2022-09-GSAFAC-0000060904 (Auditor Certified status)
https://app.fac.gov/admin/audit/singleauditchecklist/60958/change/?_changelist_filters=q%3DQN8HF1D7C9L8

@danswick @Leighdiddy Should we reach out to see if they are having issues or assume that they are still working on the new report?

**Comment by James-Paul-Mason** [comment]
Blocked pending clarification on next steps and end state of this ticket.

**Comment by James-Paul-Mason** [comment]
Attaching [this older excel validation waiver tracking sheet](https://docs.google.com/spreadsheets/d/1bJA3jY4Y2MqHtaX2DOuk1SHGsMOWTkxWDIG22wLZbPU/edit?gid=0#gid=0) as part of documentation. I think we decided we will just use ZD tags to track validation waivers. I did further research with our ZD representative and discovered tags are easily searchable and in fact very reliable. [This document on CX success stories](https://docs.google.com/document/d/12k0XluZXU23UNGmZ9b44KQmMKMJJ96OW8q4nk7RJpgw/edit) demonstrates how to tag a ticket and search for it.
